src/App/shared/components/Footer.js

```diff
@@ -21,24 +32,43 @@ const FooterBox = styled(Box)`
     a:hover {
       color: #ffc43c;
     }
+
+    .ui.image {
+      width: 100px;
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      right: 15px;
+      margin: auto;
+    }
   }
 `;
 
 function Footer() {
   return (
     <FooterBox>
-      <Link to="https://pages.semanticscholar.org/coronavirus-research">
-        COVID-19 Open Research Dataset (CORD-19)
-      </Link>
-      . 2020. Version 2020-03-27. Accessed 2020-03-28.{' '}
-      <Link to="https://doi.org/10.5281/zenodo.3727291">
-        doi:10.5281/zenodo.3727291
-      </Link>
-      <br />
-      Copyright Verizon Media 2020 Licensed under{' '}
-      <Link to="https://github.com/vespa-engine/cord-19/blob/master/LICENSE">
-        Apache License 2.0
-      </Link>
+      <Segment.Group horizontal inverted>
+        <Segment className='sides'/>
+        <Segment>
+          <Link to="https://pages.semanticscholar.org/coronavirus-research">
+            COVID-19 Open Research Dataset (CORD-19)
+          </Link>
+          . 2020. Version 2020-03-27. Accessed 2020-03-28.{' '}
+          <Link to="https://doi.org/10.5281/zenodo.3727291">
+            doi:10.5281/zenodo.3727291
+          </Link>
+          <br />
+          Copyright Verizon Media 2020 Licensed under{' '}
+          <Link to="https://github.com/vespa-engine/cord-19/blob/master/LICENSE">
+            Apache License 2.0
+          </Link>
+        </Segment>
+        <Segment className='sides'>
+          <Link to="/">
```

Comment:
This could link to vespa.ai ?

Reply:
Good call, @freva. I'll add this in a follow-up PR.